kernel/Kconfig

```
@@ -10,9 +10,11 @@ module-str = kernel
 source "subsys/logging/Kconfig.template.log_config"
 
 config MULTITHREADING
-	bool "Multi-threading"
+	bool "Multi-threading (DEPRECATED)"
```

How about `(DEPRECATED for disabled)` or something like that?

---

or
```suggestion
	bool "Multi-threading (Single threading in deprectated)"
```

and then add this to the Kconfig:
```
if ! MULTITHREADING                                                                                 
comment "*** WARNING ***"                                                                           
comment "Single threading is deprecated (MULTITHREADING disabled)"                                  
endif            
```

Will result in the following: